doc/book/app_server/cookbook.rst

```diff
@@ -776,10 +776,8 @@ to turn Tarantool into a web server.
         return self:render{ json = { ['Your-IP-Is'] = self:peer().host } }
```

Comment:
`self:peer().host` should be replaced with `self.peer.host` for http v1. Let's start the script and test it with curl:

```shell
$ curl http://localhost:8080
```

You'll see the error with current variant of the code.

Reply:
Seems resolved.